src/components/Table/utils/TableState.ts

```diff
@@ -53,6 +53,7 @@ export class TableState {
   private initialSortState: SortState | undefined;
   // The server-side `onSort` callback, if any.
   private onSort: ((orderBy: any | undefined, direction: Direction | undefined) => void) | undefined;
+  onRowSelect: OnRowSelect<any> | undefined = undefined;
```

Comment:
**thought (non-blocking):** Do you think it'd be worth giving `TableState` the actual `Row` kind generic vs always having it be `any`?

Reply:
That ends up happening in the next PR :-)